### PR TITLE
chore(build): automate build process

### PR DESCRIPTION
**Pull request by elfrank**
## Brief Description
**TL;DR:** `npm run release` creates new releases.

incorporate release-it (https://www.npmjs.com/package/release-it) to
manage our builds, github tagging and github releases.

**creating a build**
Run `npm run release`: this will kick-off the `release.sh` script. It
expects a `GITHUB_TOKEN` located at a local `.env` file. Then, it runs
`npm install` and `npm run test`. If they succeed, then it proceeds to
create a new build by running `npm run build`.
Lastly, you will see an interactive CLI interface, where you can select
`patch`, `minor` or `major` among the options.

The result is a new git tag with the corresponding version, and a new
Github release.

## Image(s) or GIFs (if applicable)
Screen Shot 2019-10-16 at 1 52 10 PM

## Pull Request Guidelines

- [x] I have added pull requests labels which describe my contribution.
- [x] All existing tests passed.
    - [x] I have added tests to cover my changes, of which pass.
- [ ] I have [compared](https://github.com/hoverinc/ray-tracing-renderer/wiki/Contributing#comparing-changes) the render output of my branch to `master`.
- [x] My code has passed the ESLint configuration for this project.
- [ ] My change requires modifications to the documentation.
    - [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by elfrank** [comment]
@jaxry nice, let's try it out :)
